tests/e2e/lba_procedures/workloads.yaml

```diff
@@ -3,21 +3,36 @@ template_cluster: &template_cluster
     main:
       args: ["--bolt-port", "7687", "--log-level=TRACE"]
       log_file: "lba-e2e.log"
-      setup_queries: [
-        "Create (:Label1 {id: 1}) ;",
-        "Create (:Label1 {id: 2}) ;",
-        "Create (:Label1 {id: 3}) ;",
-        "Create (:Label1 {id: 4}) ;",
-        "Create (:Label1 {id: 5}) ;",
-        "Create (:Label1 {id: 6}) ;",
-        "Create (:Label2 {id: 1}) ;",
-        "Create (:Label2 {id: 2}) ;",
-        "Create (:Label2 {id: 3}) ;",
-        "Create (:Label2 {id: 4}) ;",
-        "Create User Josip ;",
-        "Create User Boris ;",
-        "Grant Read On Labels :Label1 to Boris;",
-      ]
+      setup_queries:
+        [
+          "Create (:Label1 {id: 1}) ;",
+          "Create (:Label1 {id: 2}) ;",
+          "Create (:Label1 {id: 3}) ;",
+          "Create (:Label1 {id: 4}) ;",
+          "Create (:Label1 {id: 5}) ;",
+          "Create (:Label1 {id: 6}) ;",
+          "Create (:Label2 {id: 1}) ;",
+          "Create (:Label2 {id: 2}) ;",
+          "Create (:Label2 {id: 3}) ;",
+          "Create (:Label2 {id: 4}) ;",
+          "Create User Josip ;",
+          "Create User Boris ;",
+          "Grant Read On Labels :Label1 to Boris;",
+        ]
+      validation_queries: []
+
+read_permission_queries: &read_permission_queries
+  cluster:
+    main:
+      args: ["--bolt-port", "7687", "--log-level=TRACE"]
+      log_file: "lba-e2e.log"
+      setup_queries:
+        [
+          "CREATE USER admin IDENTIFIED BY 'test';",
+          "GRANT ALL PRIVILEGES TO admin;",
+          "CREATE USER user IDENTIFIED BY 'test';",
+          "GRANT ALL PRIVILEGES TO user;",
+        ]
```

Comment:
Same as above, `[]` should only be used for the one-line list in YAML (https://yaml.org/spec/1.2.2/#21-collections)